internal/cmd/list.go

```diff
@@ -46,39 +51,24 @@ func list(cli *CLI) error {
 			continue
 		}
 
-		resource := api.FormatResourceURN(g.DestinationName, g.DestinationResource)
-		if grantsByResource[resource] == nil {
-			grantsByResource[resource] = make(map[string]struct{})
-			resources = append(resources, resource)
-		}
-
-		grantsByResource[resource][g.Privilege] = struct{}{}
-	}
-	sort.Strings(resources)
-
-	type row struct {
-		Name   string `header:"NAME"`
-		Access string `header:"ACCESS"`
+		rows = append(rows, row{
+			Name:     g.DestinationName,
+			Resource: g.DestinationResource,
+			Access:   g.Privilege,
+		})
 	}
 
-	var rows []row
-	for _, k := range resources {
-		v, ok := grantsByResource[k]
-		if !ok {
-			// should not be possible
-			return fmt.Errorf("unexpected value in grants: %s", k)
+	sort.SliceStable(rows, func(i, j int) bool {
```

Comment:
This changes the output slightly. Instead of aggregating access for a particular destination/resource to a single row, this keeps it simple and gives each grant their own row.